src/components/ArticleList.jsx

```diff
@@ -19,14 +19,21 @@ const ArticleList = () => {
     fetchArticleList();
   }, []);
 
-  let articleCards = articleList.map((article) => {
-    return (
-      <ArticleCard
-        article={article}
+  let filteredArticles = () => {
+    switch (category) {
+      case "":
+        return articleList;
+      case "current":
+        return articleList.filter((article) => {
+          return Date.now() - Date.parse(article.published_at) < 86400000;
```

Comment:
86400000 milliseconds is equal to 24h, for anyone reading this wondering. I just googled it.